index.d.ts

```diff
@@ -26,8 +26,20 @@ export declare class ActionsObservable<T extends Action> extends Observable<T> {
   ofType<R extends T = T>(...key: R['type'][]): ActionsObservable<R>;
 }
 
+export declare class StateObservable<S> extends Observable<S> {
+  source: Subject<S>;
+  constructor(stateSubject: Subject<S>, store: Store<S>);
+
+  value: S
+
+  // deprecated
+  getState(): S
+  dispatch: Store<S>["dispatch"]
+}
+
+
 export declare interface Epic<T extends Action, S, D = any, O extends T = T> {
-  (action$: ActionsObservable<T>, store: MiddlewareAPI<S>, dependencies: D): Observable<O>;
```

Comment:
Can you rename the parameter to state$?

Reply:
I tried making this change myself but it looks like you don't have the option checked to "allow edits from maintainers". https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Reply:
Hm, that's strange - I think that option is enabled so I wonder if there is another issue. First pull request so I'm on unfamiliar ground!

Anyway, I think this is now fixed.

Reply:
This change doesn't remove the import of `MiddlewareAPI` which is no longer used.